docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to create a stream worker that reads data added to a collection and sends the data to a stream. After you create the stream worker, there are instructions for testing it. 
```

Comment:
Change to: In this tutorial you will write your first stream worker. Once published, the stream worker will listen for new data to be written to a collection, and subsequently write that data to a stream. For an additional challenge, you could extend this stream worker to grab the data from the stream, process/transform the data (like finding an average over x events), and write it to a new collection, or even trigger a different downstream event or action.

Reply:
We might put the additional challenge at the end, sort of next steps, perhaps.

Reply:
@dallas-macrometa @diana-macrometa How is this as a slight rewrite of Dallas's suggestion?

```
This page guides you through creating your first stream worker. The stream worker in this example automatically creates a collection and a stream. Once published, the stream worker listens to the collection for new writes and adds them to the stream.
```